### PR TITLE
Don't start service dependencies when running commands

### LINKED ISSUE #389: Ignore dependencies when running service command
At the moment, commands can define their service dependencies and Sihl makes sure to start the services in the right order.
This is too complex and it doesn't always do the right thing, so the command implementer has to make sure that the necessary services are started.

### PR DESCRIPTION
**Pull request by jerben**
Fix #389

### CODE REVIEW

File: sihl/src/core_command.ml [hunk 1]
```diff
@@ -85,6 +85,8 @@ let run commands args =
     in
     (* TODO catch all exceptions here *)
     command.fn rest_args
+  (* TODO [jerben] Print time to execute functions, "Command ran successfully
```
Comment (aronerben) [inline sihl/src/core_command.ml:88]: Please implement now if possible.

File: sihl-user/src/sihl_user.ml [hunk 2]
```diff
@@ -181,23 +181,25 @@ module Make (Repo : User_repo.Sig) : Sihl.Contract.User.Sig = struct
       else Lwt_result.fail IncorrectPassword
   ;;
 
+  let start () = Lwt.return ()
```
Comment (aronerben) [inline sihl-user/src/sihl_user.ml:184]: Please also do this for other commands (migration command for example).

**Review by aronerben — CHANGES_REQUESTED** [review]

File: sihl/src/core_command.ml (hunk 1 above)
Comment (jerben) [reply sihl/src/core_command.ml]: Done

File: sihl-user/src/sihl_user.ml (hunk 2 above)
Comment (jerben) [reply sihl-user/src/sihl_user.ml:184]: Done

Comment (aronerben) [reply sihl-user/src/sihl_user.ml:184]: Thx!

File: sihl/src/core_command.ml (hunk 1 above)
Comment (aronerben) [reply sihl/src/core_command.ml]: Thx!

**Review by aronerben — APPROVED** [review]

**Pull request merged**